src/etl/variation_etl.py

```diff
@@ -54,6 +54,19 @@ class VariationETL(ETL):
             MATCH (s:SOTerm {primaryKey:row.soTermId})
             CREATE (o)-[:VARIATION_TYPE]->(s)"""
 
+    pubs_query_template = """
+        USING PERIODIC COMMIT %s
+        LOAD CSV WITH HEADERS FROM \'file:///%s\' AS row
+            MATCH (o:Variant {primaryKey:row.variantId})
+            MERGE (p:Publication {primaryKey:row.publicationId})
+              ON CREATE SET p.pubModId = row.pubModId,
+                 p.pubMedId = row.pubMedId,
+                 p.pubModUrl = row.pubModUrl,
+                 p.pubMedUrl = row.pubMedUrl
+
+           MERGE (o)-[pn:ASSOCIATION]-(p)
```

Comment:
This is just an idea, I feel like it should point towards (o) although I am not sure if it matters - just want to get it right before people start using it. This is the same as for other queries that you have made further down. Other than that looks great.